tests/cache/tests.py

```diff
@@ -1398,6 +1398,20 @@ def test_pylibmc_options(self):
         self.assertTrue(cache._cache.binary)
         self.assertEqual(cache._cache.behaviors['tcp_nodelay'], int(True))
 
+    def test_default(self):
+        """
+        Test if default value is used when it's provided. Tests if None is
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things.

Reply:
:+1: